src/main/java/ch/uzh/ifi/hase/soprafs22/lobby/Lobby.java

```diff
@@ -29,13 +31,17 @@ public class Lobby implements ILobby {
     private byte[] qrCode;
     private Random random;
 
-    public Lobby(Long id, String name, Visibility visibility) {
+    public Lobby(Long id, String name, Visibility visibility, RegisteredUser registeredUserAsHost) {
```

Comment:
I don't understand why this is in the constructor. Why treat host different than other players?

Reply:
Because we add the host in the constructor, as every lobby must have a host, else it should get removed.
If we link the host after adding the host. The key (token) of the host is a randomly generated UUID and not the token of the registered user.
Probably there are better ways, similar to the generatePlayer method in the lobby which would be resolved when we create a Playerfactory or something like that...